lib/API/JSON/v1/AutomatedLatestContent.php

```diff
@@ -4,20 +4,22 @@
 
 use MailPoet\API\JSON\Endpoint as APIEndpoint;
 use MailPoet\Config\AccessControl;
-use MailPoet\WP\Hooks;
+use MailPoet\WP\Functions as WPFunctions;
 use MailPoet\WP\Posts as WPPosts;
 
 if(!defined('ABSPATH')) exit;
 
 class AutomatedLatestContent extends APIEndpoint {
   /** @var \MailPoet\Newsletter\AutomatedLatestContent  */
   public $ALC;
+  private $wp;
```

Comment:
Shouldn't we also add the `/** @var WPFunctions  */ comment since the previous property has it? 

And also I think we should leave one empty row between properties, we do it in most places so it should be consistent.

Reply:
We don't follow any specific rule about typehinting properties. I would vote for adding type comments to all properties, but then we will need to maintain them as part of the code!

Reply:
AFAIK PHPStan does that in one of the higher levels.